downloaded/installLisk.sh

```diff
@@ -101,17 +101,21 @@ download_lisk() {
 		return
 	fi
 
-	LISK_VERSION=lisk-$UNAME.tar.gz
+	if [[ ! $LISK_VERSION_NUMBER ]]; then
+		LISK_VERSION_NUMBER=$(curl -s "https://downloads.lisk.io/lisk/$RELEASE/" | grep '^<a .*>.*/<' | sed 's/^.*"\(.*\)\/".*$/\1/' | sort -Vr | head -n 1)
```

Comment:
This seems very fragile.
I'd much rather have a `latest.txt` file containing the version number of the newest release (see e.g. [gentoo](http://gentoo.mirrors.ovh.net/gentoo-distfiles/releases/amd64/autobuilds/latest-install-amd64-minimal.txt]))

Reply:
That does sound much better